pkg/process/checks/net.go

```diff
@@ -50,8 +50,11 @@ type ConnectionsCheck struct {
 	notInitializedLogLimit *putil.LogLimit
 	// store the last collection result by PID, currently used to populate network data for processes
 	// it's in format map[int32][]*model.Connections
-	lastConnsByPID *atomic.Value
-	probe          procutil.Probe
+	lastConnsByPID   *atomic.Value
+	probe            procutil.Probe
```

Comment:
I think this field isn't needed anymore since it got moved into `ProcData`.

Reply:
Good catch! I've removed it.